### PR TITLE
Support mixed LogisticRegression model

### PR DESCRIPTION
**Pull request by wbo4958**
Add a cpu() method to return a pyspark built-in LogisticRegressionModel

### CODE REVIEW

**Review by eordentlich — COMMENTED** [review]
Minor comment.
Looks good.

**Comment by lijinf2** [comment]
LGTM

**Comment by wbo4958** [comment]
build

File: python/src/spark_rapids_ml/classification.py [hunk 1]
```diff
@@ -842,6 +851,39 @@ def __init__(
         super().__init__(dtype=dtype, n_cols=n_cols, coef_=coef_, intercept_=intercept_)
         self.coef_ = coef_
         self.intercept_ = intercept_
+        self._lr_spark_model: Optional[SparkLogisticRegressionModel] = None
+
+    def cpu(self) -> SparkLogisticRegressionModel:
+        """Return the PySpark ML LinearRegressionModel"""
```
[the remark this answers is not in the document]
Comment (wbo4958) [reply python/src/spark_rapids_ml/classification.py]: Done, thx

**Review by leewyang — APPROVED** [review]

**Pull request merged**